django-stubs/test/testcases.pyi

```diff
@@ -50,10 +51,15 @@ class _AssertTemplateUsedContext:
     context: ContextList = ...
     def __init__(self, test_case: Any, template_name: Any) -> None: ...
     def on_template_render(self, sender: Any, signal: Any, template: Any, context: Any, **kwargs: Any) -> None: ...
-    def test(self): ...
-    def message(self): ...
-    def __enter__(self): ...
-    def __exit__(self, exc_type: Any, exc_value: Any, traceback: Any): ...
+    def test(self) -> None: ...
+    def message(self) -> str: ...
```

Comment:
Removed in Django 4.1, but previously returned a `str`: https://github.com/django/django/commit/806efe912b846c1fde250c9321d8334b7517cd56#diff-7833da5b45a68d00834388d97dd5c4413e3796497c7bc5e0cc2621b08a2d0df1L111